.github/workflows/zero-downtime-tests.yml

```diff
@@ -181,6 +177,12 @@ jobs:
           HELM_CHART_NAME=$(ls dist/keptn-installer/keptn*.tgz)
           echo "##[set-output name=HELM_CHART_NAME;]$(echo ${HELM_CHART_NAME})"
 
+      - name: Setup upgrade chart value
+        id: upgradeTo
+        run: |
+          USER_INPUT=${{ github.event.inputs.upgradeTo }}
+          echo "::set-output name=chart::${USER_INPUT:-"https://charts-dev.keptn.sh/packages/keptn-0.19.0-dev.tgz"}"
+
```

Comment:
You have the same default value provided in the pipeline input at the top. Why do you do it again here?